docker/README.md

```diff
@@ -15,18 +15,22 @@ By default, the pipeline will install a single-threaded version of [Novoalign](h
 ### To build
   Navigate to the directory containing the `Dockerfile`, then run:
   `tar -czh . | docker build --rm -`
+
   The `tar` is necessary because Docker cannot dereference symlinks, and by tarring the directory, symlinks
   to files in higher filesystem levels can be used. In particular, it is assumed a symlink exists within the directory containing the `Dockerfile` to the `easy-deploy-script/` directory of this repo.
-  
+
+  It is a good idea to use `-t` argument to give the build a tag so it is easier to manage images later. For example:
+  `tar -czh . | docker build --rm -t local/viral-ngs -`
+
   To build with a specific version of viral-ngs, a Docker `--build-arg` may optionally be specified. Ex.:
   ```
-  tar -czh . | docker build --rm --build-arg VIRAL_NGS_VERSION=1.15.0 -
+  tar -czh . | docker build -t local/viral-ngs:1.16.0 --build-arg VIRAL_NGS_VERSION=1.16.0  -
```

Comment:
Please add the `--rm` back in; it ensures intermediate containers are removed after a successful build.